CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
```

Comment:
along with this, delete https://github.com/jenkinsci/helm-charts/blob/main/code-of-conduct.md

Reply:
good catch.